### ISSUE TITLE
MDBOOK007: false error for an escaped \{{#include}}

### ISSUE THREAD

**Issue by n8behavior** [joshrotenberg/mdbook-lint#499]
MDBOOK007 reports an error for an escaped include. mdBook does not process an
escaped include, so the file does not need to exist.

**src/p.md**

```markdown
\{{#include ../file.md:name}}
```

**mdbook-lint lint --enable MDBOOK007 src/p.md**

```text
error[MDBOOK007]: Include file '../file.md' not found. Resolved path: src/../file.md
  --> src/p.md:6:2
```

mdBook renders the line as the literal text `{{#include ../file.md:name}}`. A
book uses this form to show the include syntax to a reader.

mdbook-lint 0.16.1, mdbook 0.5.4.
